src/transform/utilsFS.ts

```diff
@@ -62,11 +63,11 @@ export function getFileTokens(
 
     // Read the content only if we dont have one in the args
     if (!content) {
-        if (filesCache[path]) {
-            content = filesCache[path];
+        if (filesCache.has(path)) {
+            content = filesCache.get(path) as string;
```

Comment:
Is `as string` necessary here? Seems like the type of keys and values is set with the constructor, so TS should know this is a string.

Reply:
@main-kun need, cause without it will be string | undefined.